### ISSUE TITLE
Please remove dependencies 

### ISSUE THREAD

**Issue by max-mykhailenko** [boneskull/angular-tags#21]
Your bower file have these dependencies.

"dependencies": {
    "angular-ui-bootstrap-bower": "~0.6.0",
    "angular": "1.2.0-rc.2",
    "font-awesome": "~3.2.1",
    "bootstrap-css": "~3.0.0"
  }

I have already installed less version of bootstrap and I don't want download bootstrap-css. How I can do this?

**Comment by boneskull** [comment]
Hi,

It’s required for the templates to look right, but if you already have bootstrap there’s no reason you can’t just remove the directory.

**Comment by max-mykhailenko** [comment]
I use grunt-bower and prepare project automatically. I can't remove unnecessary folders every time, when I want to add some plugin. Maybe you have more handy solution?

**Comment by boneskull** [comment]
I’ll consult with some other members of my team, but really we want the bower package to provide everything you need to get the plugin working.  I could conceivably provide two versions: one with bootstrap and the templates, and one without the templates.

thanks,
Chris

**Comment by max-mykhailenko** [comment]
One more thing, my bower.json looks like this:
{
....
"angular": "1.2.0-rc.2",
"angular-bootstrap" : "0.6.0",
"bootstrap": "3.0.0",
"angular-tags" : "0.2.10"   // or any another  
....
}

And bower don't understand that "bootstrap-css" and my "bootstrap" it's the same things. Same situation with my "angular-bootstrap" which equivalent "angular-ui-bootstrap".

Welcome any thoughts about this.

**Comment by gigablox** [comment]
Good work on this project so far but I would like to piggyback on this idea:

Minimum viable product. Let your users worry about how pretty it's going to look. If you want it to look pretty in your demo show an example of using bootstrap to overwrite existing styles rather then bundling them with the project. 

Iconography is currently a dependency of the templates so its not exactly "optional" unless you use the `-tpls` version and swap out that icon for something else.

**Comment by boneskull** [comment]
Yeah, I agree we’d like to drop any CSS requirement.  Will be talking with one of our CSS dudes about this “soon”.  Thanks for the input!

**Comment by tastycode** [comment]
Agree... I ran into a wall with this. Using yeoman-angular, the generated project already has bootstrap-sass , and your dependency breaks everything and makes the whole app unusable. You should have a version that purely implements the logic, and allow the developer to implement CSS integration. As this is now, your package is catering too much to people trying to get it working on an empty project. Most people will be adding this into an existing project which probably already has ui bootstrap.

**Comment by boneskull** [comment]
I can do that (looks like designer won't have the time), so I'll put it on my to-do list, but feel free to gut it and submit a PR in the meantime.

-- 
Christopher Hiller

On January 13, 2014 at 6:06:47 PM, Thomas Devol (notifications@github.com) wrote:

Agree... I ran into a wall with this. Using yeoman-angular, the generated project already has bootstrap-sass , and your dependency breaks everything and makes the whole app unusable. You should have a version that purely implements the logic, and allow the developer to implement CSS integration. As this is now, your package is catering too much to people trying to get it working on an empty project. Most people will be adding this into an existing project which probably already has ui bootstrap.

**Comment by konsumer** [comment]
Same problem with brunch, using bootstrap-less.
